client/src/utils/constants.js

```diff
@@ -1,10 +1,10 @@
 export const opportunityType = [
-  { key: "1", text: "voluntary work", value: "voluntary work" },
-  { key: "2", text: "training", value: "training" },
-  { key: "3", text: "internship", value: "internship" },
-  { key: "4", text: "work experience", value: "work experience" },
-  { key: "5", text: "part time job", value: "part time job" },
-  { key: "6", text: "full time job", value: "full time job" }
+  { key: "1", text: "Volunteer", value: "Volunteer" },
```

Comment:
why do you need `key`? you can use `value` as the key `.map` as it is unique

Reply:
to insert the array to the dropdpwn as option should be in that form
https://react.semantic-ui.com/modules/dropdown/#types-selection